test/e2e_appium/pages/onboarding/loading_page.py

```diff
@@ -0,0 +1,40 @@
+"""
+Loading Page for Status Desktop E2E Testing
+
+Page object for splash during onboarding.
+"""
+
+import logging
+from ..base_page import BasePage
+from locators.onboarding.loading_screen_locators import LoadingScreenLocators
+
+
+class LoadingPage(BasePage):
```

Comment:
```suggestion
class SplashScreen(BasePage):
```
easier to understand , closer to the namings in QML